ha/setup/ha_setup.py

```diff
@@ -855,6 +855,16 @@ def main(argv: list):
             log_path = Conf.get(const.HA_GLOBAL_INDEX, f"LOG{_DELIM}path")
             log_level = Conf.get(const.HA_GLOBAL_INDEX, f"LOG{_DELIM}level")
             Log.init(service_name='ha_setup', log_path=log_path, level=log_level)
+        elif sys.argv[1] == "cleanup":
+            if not os.path.exists('const.HA_CONFIG_FILE'):
+                a_str = f'Cleanup can not be proceed as \
```

Comment:
Can we change error to warning 

Also is return code 0 or 1

Reply:
Sure. Will change.

Reply:
Done.